package.json

```diff
@@ -52,9 +52,9 @@
     "@typescript-eslint/parser": "^4.7.0",
     "@typescript-eslint/typescript-estree": "^4.7.0",
     "cookie-parser": "^1.4.4",
+    "copy-webpack-plugin": "^9.0.0",
     "copyfiles": "^2.1.1",
     "cors": "^2.8.5",
-    "cpy-cli": "^3.1.1",
```

Comment:
Removed this utility and replaced it with the webpack plugin above. This one had a dependency that had a security alert.